### ISSUE TITLE
Running direct from War, fails to find "config", unable to create first page in wiki

### ISSUE THREAD

**Issue by faceless2** [gollum/gollum#2080]
Running brand new Gollumn download from the WAR, I am unable to create any pages due to internal error.

**To Reproduce**
1. Download the War from the releases page (I'm testing 6.0.0)
2. Create a new git repository in the current directory with `git init`
3. Start gollum in the current directory with `java -jar gollum.war -S gollum -p 8081`
4. Visit localhost:8081 - I see the gollum wiki, but when I try to save I get a 500 Internal Error, and the console gives me a (very) long trace ending in `Java::JavaIo::IOException - No such file or directory - /tmp/jruby404533616076248845extract/config`

Which is correct as file doesn't exist, although there are ` config.rb` and `config.ru` files in that folder. I've also tried adding `--config /abs/path/to/config.rb` as an argument, but it makes no difference.

**Environment Info**
This is on Linux, OpenJDK 21.0.3
```
$ java -jar gollum.war -S gollum --version               
Gollum 6.0.0
```

**Comment by step-** [comment]
Same error here on Linux with JDK 23.

**Comment by step-** [comment]
After reading #2057 I was able to make it work with the following command:

    java -jar /path/to/gollum-6.0.0.war -S gollum --ref main /tmp/wikitest

It seems the War file wants absolute paths.

**Comment by dometto** [comment]
Thanks for opening this issue. I don't know if there is a fix for #2057, if anyone has any thoughts they're most welcome. I've added information about using absolute paths to the README and the wiki. See e06121f914207c5e5af9cf82f1334c7afce3433d
